mount-daemon/version/version.go

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 Hewlett Packard Enterprise Development LP
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package version
+
+// The current clientmountd version string.
+var (
+	// Version contains the current version of the clientmount daemon.
+	version = "v0.0.1"
```

Comment:
We'll eventually need to do the same with version and probably use git tags for this, right?

Reply:
See how we use git-version-gen in the Makefile.
And see https://dataworkflowservices.github.io/repo-guides/create-a-release/readme/ for more info about how that's used to make releases.